test/LogNormal/unit/Swap.t.sol

```diff
@@ -91,7 +90,7 @@ contract LogNormalSwapTest is LogNormalSetUp {
 
         console2.log(invariant);
 
-        uint256 amountOut = preReserves[0] - rx;
+        uint256 amountOut = rX - rx;
```

Comment:
We might want to rename these variables, that's not super explicit haha.

Reply:
Agreed could be good.